src/boot/msg.ml

```diff
@@ -41,6 +41,11 @@ type message = id * severity * info * arguments
 exception Error of message
 
 
+let eq_info a b = match a, b with
+  | NoInfo, NoInfo -> true
+  | Info(f1, l11, c11, l21, c21), Info(f2, l12, c12, l22, c22) ->
+     l11 = l12 && c11 = c12 && l21 == l22 && c21 == c22 && f1 =. f2
```

Comment:
Is there a motivation for comparing the starting coordinates with `=` and the latter with `==`?

Reply:
No, it's a typo. Thanks.